src/client/core/utils/dom.js

```diff
@@ -443,3 +443,9 @@ export function findIframeByWindow (iframeWindow, iframeDestinationWindow) {
 
     return null;
 }
+
+export function canAcceptValue (element) {
```

Comment:
There is not special term in HTML spec that fits well for this meaning, e.g. see [textarea categories](https://html.spec.whatwg.org/#the-textarea-element). However `isEditorElement` sounds good to me